src/web/utils/treeUtils.ts

```diff
@@ -43,16 +43,22 @@ function addNodeToTree(element: HTMLMarkdownElement, parentTreeNode: TreeNode, t
 
 function updateTreeElementRefs(treeRoot: TreeNode, element: HTMLMarkdownElement) {
   const stack: TreeNode[] = [treeRoot];
+  const treeElements = element.querySelectorAll('[data-id], [data-id] *') as NodeListOf<HTMLMarkdownElement>;
```

Comment:
```suggestion
  const treeElements = element.querySelectorAll('[data-id]') as NodeListOf<HTMLMarkdownElement>;
```

Why we need `[data-id] *`? All elements that can be found in tree have `data-id`